core/ext/mod.rs

```diff
@@ -28,3 +28,8 @@ impl ExtFunc {
         }
     }
 }
+
+pub fn init(db: &mut crate::Database) {
```

Comment:
just to be clear we are only moving the `uuidv4` function into a dynamically loaded extension in this PR, and the rest stay where they are?

Reply:
Yes. That's because I wanted to keep this PR small but have at least one show-case of how the new API is used. Or I am lazy and hope other people will fix things up once this lands in main.